packages/extension/src/lib/CustomRouter.ts

```diff
@@ -54,7 +55,7 @@ export default class CustomRouter implements NextRouter {
   }
 
   async replace(url: Url): Promise<boolean> {
-    window.location.href = url as string;
+    window.location.href = typeof url === 'object' ? formatUrl(url) : url;
     return true;
```

Comment:
We don't need this changes anymore since we redirect to app?

Reply:
Not really

Reply:
Should I revert it?

Reply:
I would say yes, just to minimize code changes

Reply:
Removed.